using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -170,8 +190,14 @@ ACPCampaign.resetLinkageFields();
 
 ## Set up push messaging
 
+To enable push messaging with Adobe Campaign, the push identifier received from APNS (Apple Push Notification Service) or FCM (Firebase Cloud Messaging) must be sent to the Adobe Identity service by invoking `ACPCore::setPushIdentifer`. This API should also be invoked if there is any change in the push identifier (for example when a new token is generated due to the previous token being expired). After the push identifier is sent to the Identity service, Campaign will then retrieve the push identifier from the Identity service directly.
+
+For more detailed information on setting up your iOS app to connect to APNS for retrieving a device token to be used as a push identifier, see https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns?language=objc.
```

Comment:
Please reword the sentence as follows:
"For more information about setting up your iOS app to connect to APNS to retrieve a device token that can be used as a push identifier, see https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns?language=objc."